src/Protocol/ActivityPub/Processor.php

```diff
@@ -650,7 +651,7 @@ private static function processContent($activity, $item)
 
 				$content = self::removeImplicitMentionsFromBody($content, $parent);
 			}
-			$item['content-warning'] = HTML::toBBCode($activity['summary']);
+			$item['content-warning'] = (!empty($activity['summary']) ? HTML::toBBCode($activity['summary']) : '');
```

Comment:
```suggestion
			$item['content-warning'] = HTML::toBBCode($activity['summary'] ?? '');
```

Reply:
The same here. It would execute a lot code for an empty string.